usr/share/rear/lib/layout-functions.sh

```diff
@@ -576,3 +576,15 @@ blkid_label_of_device() {
     echo "$label"
 }
 
+# Returns 1 if the device is an LVM physical volume
+# Returns 0 otherwise or if the device doesn't exists
+is_disk_a_pv() {
+    disk=$1
+#   grep "^lvmdev .* ${disk} " $VAR_DIR/layout/disklayout.conf > /dev/null 2>&1
+    grep "^lvmdev .* ${disk} " $VAR_DIR/layout/disklayout.conf
```

Comment:
please use `$LAYOUT_FILE` instead of `$VAR_DIR/layout/disklayout.conf`
and, why not using `-q` option to avoid displaying to the standard output?
I prefer something like:
````
if grep -q "^lvmdev .* ${disk} " $LAYOUT_FILE ; then
   return 1
else
   return 0
fi
````